test_results.json

```diff
@@ -3,1762 +3,2010 @@
     {
       "Author": "plusuncold",
       "Language": "C++",
-      "Word": "pax, kivu, zax, bock, big",
-      "Score": "12, 11, 19, 12, 6",
-      "Time (ms)": 18.4,
+      "Word": "dwarf, hin, zo, jud, ju",
```

Comment:
test_results.json will be updated in standardized droplet after your PR is merged.